packages/odyssey-stylelint/src/index.js

```diff
@@ -37,219 +38,6 @@ module.exports = {
     'declaration-no-important': true,
     'max-nesting-depth': [ 3, { ignoreAtRules: [ 'media', 'supports', 'include' ] } ],
     'order/properties-alphabetical-order': null,
```

Comment:
I tried removing this, but it seems that all other property order rules are ignored without this flag set. Oddly, alpha sort _does not_ offer auto-fixing.